src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -236,6 +237,32 @@ public void parseRecognizesEntryWithUnknownType() throws IOException {
     }
 
     @Test
+    public void parseReallyUnknownType() throws Exception {
+        String bibtexEntry = "@ReallyUnknownType{test," + OS.NEWLINE +
+                " Comment                  = {testentry}" + OS.NEWLINE +
+                "}";
+
+        Collection<BibEntry> entries = new BibtexParser(importFormatPreferences).parseEntries(bibtexEntry);
+        BibEntry entry = entries.iterator().next();
+
+        String resourceName = "/testbib/reallyunknowntype.bib";
```

Comment:
I've wanted to test explicit against the testfile "reallyunknowntype.bib". If I see right, the other tests don't use such test files.